contracts/bridge/BridgeTokens.sol

```diff
@@ -65,6 +68,7 @@ contract BridgeTokens is Ownable {
         delete allowedTokens[_token];
         delete lockedTokens[_token];
 
+        // This iteration consumes 818 gas. 5,000,000 gas limit is enough for the iteration of maxTokenCnt.
```

Comment:
It would be better if you specify the value of `allowedTokenList.length` consuming 818 gas.

Reply:
Got it. How about this?
```suggestion
        // This iteration consumes 818 gas. 81800 gas is needed for the maxTokenCnt iteration.
```

Reply:
Looks good. Thank you.

Reply:
reflected : ) Thanks.